pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -122,6 +122,8 @@ type Receiver struct {
 	WeChatConfigs []WeChatConfig `json:"wechatConfigs,omitempty"`
 	// List of Email configurations.
 	EmailConfigs []EmailConfig `json:"emailConfigs,omitempty"`
+	// List of VictorOps configurations.
+	VictorOpsConfigs []VictorOpsConfig `json:"victorOpsConfigs,omitempty"`
```

Comment:
To be compliant with #3690 

```suggestion
	VictorOpsConfigs []VictorOpsConfig `json:"victoropsConfigs,omitempty"`
```